src/senselab/audio/tasks/speech_to_text/api.py

```diff
@@ -22,16 +29,20 @@ def transcribe_audios(
         model (SenselabModel): The model used for transcription.
         language (Optional[Language]): The language of the audio (default is None).
         device (Optional[DeviceType]): The device to run the model on (default is None).
+        **kwargs: Additional keyword arguments to pass to the transcription function.
 
     Returns:
         List[ScriptLine]: The list of script lines.
-    """
-    if isinstance(model, HFModel):
-        return HuggingFaceASR.transcribe_audios_with_transformers(
-            audios=audios, model=model, language=language, device=device
-        )
-    else:
-        raise NotImplementedError("Only Hugging Face models are supported for now.")
-
 
-transcribe_audios_pt = pydra.mark.task(transcribe_audios)
+    Todo:
+        - Include more models (e.g., speechbrain, nvidia)
+    """
+    try:
+        if isinstance(model, HFModel):
+            return HuggingFaceASR.transcribe_audios_with_transformers(
+                audios=audios, model=model, language=language, device=device, **kwargs
```

Comment:
You pass `**kwargs` to `transcribe_audios_with_transformers` but that function does not take `**kwargs` as an input. As the parameters for the function are limited, and we have control over that function, I wonder if we should write them out specifically as parameters for this function as well? Or otherwise we should point to that function in the documentation of this one so users know what can go into `**kwargs` here.

Reply:
this is on purpose. `**kwargs` in the general api helps accepting all kind of params for the different models, and then the internal function (`transcribe_audios_with_transformers`) does the validation of the params, which is why it doesn't include `**kwargs`. the accepted params are documented in `transcribe_audios_with_transformers`.

Reply:
What is the main entry point that should be used for speech to text? The `api` documentation does not mention `transcribe_audios_with_transformers`, so if somebody uses that function as their interface for speech to text, they won't know what options are available for `**kwargs`.